app/kuma-dp/pkg/dataplane/accesslogs/streamer.go

```diff
@@ -134,8 +134,6 @@ func (s *accessLogStreamer) streamAccessLogs(reader *bufio.Reader) error {
 					continue
 				}
 			}
-
-			accessLogMsg = append(accessLogMsg, '\n')
```

Comment:
Won't it break any integration user might have? Like Dynatrace agent fetching logs and looking for some specific format?

Reply:
Added UPGRADE.md section, please take a look